docs/experiment/guides/troubleshooting/variant-jumping.md

```diff
@@ -0,0 +1,94 @@
+
+## Normal variant jumping
+
+Normal variant jumping may occur due to:
+
+1. [Targeting changes](#targeting-changes): changes to targeting rules while your experiment is running
+2. [Anonymous identity merging](#anonymous-identity-merging): anonymous users, bucketed by amplitude ID may receive different variants until they are eventually resolved via a matching user ID.
+
+### Targeting changes
+
+Performing the following actions may cause a user to jump variants:
+
+1. Adding or removing a variant
+2. Changing variant distribution weights
+3. Targeting dynamic cohorts
+4. Changing the bucketing key
+5. Updating mutual exclusion
+
+!!!tip "Avoid variant jumping by enabling sticky bucketing"
+    Enabling sticky bucketing before making targeting changes will avoid variant jumping. However, sticky bucketing may cause a [sample ratio mismatch (SRM)](./sample-ratio-mismatch.md).
+
+### Anonymous identity merging
+
+Variant jumping caused by anonymous identity merging may occur due to bucketing by Amplitude ID, when user profiles with distinct device IDs (resolved to different amplitude IDs) are merged when resolved to a common user ID. This may happen if a user uses your app on different devices, or if, on logout, the device ID is re-generated.
+
+!!!info "[Learn more about Amplitude's identity resolution and merging users.](https://help.amplitude.com/hc/en-us/articles/115003135607-Track-unique-users-in-Amplitude#h_e9913ce1-549a-4d88-a832-1f56aede581a)"
+
+ Identifying this type of variant jumping is easy by identifying the the assignment event where the user jumped between variants, and comparing the value of the Amplitude ID for both events. If the Amplitude ID is different on the two events, then there then it is very likely that anonymous identity merging was the cause.
+
+To combat this type of variant jumping, consider bucketing by:
+
+* User ID: If you're only targeting user who are logged in, and have a user ID.
+* Device ID: If you're only targeting anonymous users (e.g. sign up experiment).
+
+## Abnormal variant jumping
+
+Abnormal variant jumping is unexpected variant jumping that can't be explained by any [normal means](#normal-variant-jumping). Abnormal variant jumping can be tough to track down, however almost all abnormal variant jumping is caused by some for of identity mismatch: **when the user identity used in assignment is different from the identity used to track the exposure.** In short, abnormal variant jumping is almost always due to an inconsistency in implementation.
+
+The following examples are by no means exhaustive, but should hopefully get you thinking about the use of identity in your system with respect to Amplitude Experiment.
+
+### Multiple logged in accounts on a single device
+
+Consider this timeline for a user with multiple accounts for your app on a single device.
```

Comment:
Please define "U1" and "U2" in the intro. Something like, 

_Consider this timeline for a user with multiple accounts (U1 and U2)..._

Reply:
I realize this sounds silly, but it took me a few reads to realize it referred to the user, and we should try to reduce the chances of confusion where we can.